modules/lang-painless/src/test/java/org/elasticsearch/painless/DivisionTests.java

```diff
@@ -413,4 +413,10 @@ public void testCompoundAssignmentByZero() {
             exec("def x = 1; x /= 0; return x;");
         });
     }
+
+    public void testNoConflictWithRegex() {
+        assertEquals(1, exec("int d = 1; return d/1;"));
+        assertEquals(1, exec("int g = 1; return g/1;"));
+        assertEquals(1, exec("int j = 1; return j/1;"));
+    }
```

Comment:
We want to be extra paranoid here that the compiler doesn't see `g/` as the key for a `grok` regex when it is really division. It doesn't, but let's add an explicit test to make sure it never does.